// anaplan_remote/customerHeroMarkup.js
var customerHeroMarkup = function ( customerHeroObj ) { 
	let headStyle = `<style>
  h1 {
    color: #0b2265;
    font-weight: 400;
  }
  .single-resource {
    margin: 0 auto !important;
  }
  .collapse,
  .collapsing {
    margin-top: 1rem;
    float: left;
  }
  a[data-toggle=collapse],
  .collapse,
  .collapsing {
    font-size: 1.6rem;
    line-height: 2.4rem;
    color: #6c7782;
  }
  .customer-story.hero {
    margin-top: 10rem !important;
    padding: 7rem;
    height: 45rem;
    background: url("` + customerHeroObj.bgImage + `") no-repeat;
    background-size: cover;
  }
  .customer-story .company-logo img {
    max-width: 10rem;
  }
  .company-info h1,
  .content-type h4,
  .company-info .video {
    margin: 0 auto;
    margin-top: 2rem;
    display: block;
    font-weight: 400;
    color: #ffffff;
  }
  sup {
    color: #ffffff;
  }
  .company-info .video {
    display: inline-block;
    margin-right: 2rem!important;
  }
  .content-type h4 {
    color: #92d400 !important;
  }
  .company-info h1 {
    margin-top: 1.4rem;
    font-size: 4rem;
    line-height: 4.5rem;
  }
  #hero-img h4,
  #highlights h4 {
    font-weight: 400;
    color: #0b2265;
  }
  #hero-img h4 {
    font-weight: 700;
    text-transform: uppercase;
  }
  .company-info .play-button {
    max-width: 1.6rem !important;
    vertical-align: middle;
  }
  #highlights {
    background-color: #99a4ae;
  }
  .customer-story.highlights p {
    font-size: 3rem;
    line-height: 3.6rem;
    color: #ffffff;
  }
  .customer-story.highlights p,
  .customer-story.highlights h4 {
    text-align: center;
  }
  .customer-story.summary p,
  .customer-story.videos p  {
    margin: 3rem auto;
    font-size: 1.6rem !important;
    line-height: 2.4rem !important;
    color: #6c7782;
    width: 95%;
    float: inherit;
  }
  .customer-story.videos p {
    margin: 3rem auto 1rem;
  }
  .customer-video.center {
    margin: 0 auto;
    padding: 0;
    float: inherit;
  }
  .customer-video.center p {
    margin: 3rem auto;
    padding: 0;
    width: 100%;
  }
  .customer-story.summary h1 {
    color: #0b2265;
    font-weight: 700;
  }
  .customer-story .customer-video {
    padding-top: 3rem;
  }
  .customer-story.videos h1 {
    width: 95%;
  }
  .quote-content {
    padding: 6rem 0 0 0;
    height: 45rem;
  }
  .quote-content p,
  .quote-content strong {
    font-size: 1.6rem;
    line-height: 2.4rem;
    color: #ffffff!important;
  }
  .quotations {
    margin: 0 auto;
    padding: 0;
    height: 42rem;
    background-color: #3d7edc;
  }
  .blockquote {
    margin: 3rem auto;
    margin-top: 0;
    quotes: "\\201C""\\201D""\\2018""\\2019";
  }
  .blockquote:before {
    color: #fff;
    content: open-quote;
    font-size: 4rem;
    line-height: 0.1rem;
    margin-right: 0.25rem;
    vertical-align: -0.4rem;
  }
  .blockquote:after {
    color: #fff;
    content: close-quote;
    font-size: 4rem;
    line-height: 0.1rem;
    margin-right: 0.25rem;
    vertical-align: -0.4rem;
  }
  .blockquote h3 {
    display: inline;
  }
  .carousel-indicators li {
    width: 1.2rem;
    height: 1.2rem;
    margin: 0.4rem 0.6rem;
  }
  .carousel-indicators .active {
    width: 1.4rem;
    height: 1.4rem;
    margin: 0.3rem 0.5rem;
  }
  #quotation-carousel {
    margin: 0 auto;
    height: 42rem;
  }
  #summary {
    padding: 6rem 0 2rem;
  }
  #videos {
    padding: 0 0 6rem 0;
  }
  #other-resources h1 {
    margin-bottom: 3rem;
  }
  .quote-content {
    margin: 0 auto;
    float: inherit;
    width: 60%;
    max-width: 96rem;
  }
  .quote-content h3 {
    margin: 3rem auto;
    margin-top: 0;
    font-size: 3rem;
    line-height: 3.6rem;
    color: #ffffff;
  }
  .quote-quote {
    margin-top: 30px;
  }
  .quote-quote em {
    color: #637983 !important;
  }
  .quote-quote p {
    margin: 0 auto;
    margin-top: 14px;
    margin-bottom: 20px;
    width: 95%;
    float: right;
  }
  .quote-quote h3 {
    font-size: 14pt !important;
    line-height: 20pt !important;
    font-weight: 500;
    margin: 0 auto;
    width: 100%;
    text-align: right !important;
    float: right !important;
  }
  .customer-video .playbutton {
    margin: 0 auto;
    position: absolute;
    padding-left: 40%;
    padding-top: 15%;
    background-color: rgba(0,0,0,0.35);
    width: 100%;
    height: 100%;
    z-index: 999;
    transition: background-color 0.25s ease-in;
  }
  .customer-video .playbutton img {
    position: absolute;
    top: -10rem;
    transition: top 0.25s ease-in;
  }
  .customer-video .playbutton.active img {
    position: absolute;
    top: 8rem;
    margin-left: 1rem;
    max-width: 7rem;
  }
  .customer-video .playbutton:hover {
    background-color: rgba(0,0,0,0.75);
  }
  @media screen and (min-width: 1600px) {
    .customer-story.hero {
      background-position-x: 0;
    }
  }
  @media screen and (min-width: 992px) {
    .single-resource {
      margin: 0 auto !important;
      padding: 0 !important;
    }
    .customer-story .customer-video {
      padding-right: 3rem;
    }
  }
  @media screen and (max-width: 992px) {
    .quote-content h3 {
      font-size: 2.2rem;
      line-height: 2.8rem;
    }
    .customer-video .playbutton img {
      position: absolute;
      top: 5rem !important;
    }
    .blockquote:before,
    .blockquote:after {
      vertical-align: -0.8rem;
    }
    .quote-quote {
      margin-top: 0px;
    }
    .customer-story.hero {
      margin-top: 0 !important;
      padding: 6rem 0;
      height: 50rem;
    }
    .customer-story.summary p, .customer-story.videos p {
      width: 100%;
    }
  }
  @media screen and (max-width: 767px) {
    .speaker-pic {
      float: none;
    }
    .quotations,
    #quotation-carousel {
      height: 50rem;
    }
  }
  @media screen and (max-width: 480px) {
    .customer-story.hero {
      height: 55rem;
    }
    .quotations, #quotation-carousel {
      height: 55rem;
    }
  }
</style>`;
	let markup = `<section id="hero-img" class="customer-story hero"><div class="container"><div class="row"><div class="col-md-8 left"><div class="company-logo"><img src="` + customerHeroObj.companyLogo + `" alt="` + customerHeroObj.companyName + ` logo"/></div><div class="content-type"><h4>` + customerHeroObj.contentType + `</h4></div><div class="company-info"><h1>` + customerHeroObj.companyInfo + `</h1>`;
	if ( customerHeroObj.companyVideos.length >= 1 ) {
		markup += `<a href="#videos" class="video">Watch video <img src="/wp-content/uploads/2016/04/play_button_large_90x90.png" class="play-button" /></a>`;
	}
	if ( customerHeroObj.companyDocument !== false ||  customerHeroObj.companyDocument !== "" ) {
		markup += `<a href="` + customerHeroObj.companyDocument + `" class="video" target="_blank">Read full story <i class="fa fa-angle-right"></i></a>`;
	}
	markup += `</div>
      </div>
    </div>
  </div>
</section>`;
	return headStyle + markup;
};

module.exports = customerHeroMarkup;
