// anaplan_remote/sessionsPage.js
"use strict";

const sessionsPage = (init, modal, scripts) => {
  return `<style>
  p a.video-playlist {
    display:none!important;
  }
  #bg-image {
    z-index: -1;
  }
  #bg-content {
    transform: translateY(0);
    -webkit-transform: translateY(0);
    -ms-transform: translateY(0);
    margin-top: 60px;
  }
  .playbutton {
    width: 100%;
    height: 100% !important;
    position: absolute;
    display: block;
    left: 0;
    overflow: hidden;
  }
  .playbutton img {
    position: relative;
    top: 30px;
    max-width: 60px;
    -moz-transition: all 0.15s ease-in;
    -o-transition: all 0.15s ease-in;
    -webkit-transition: all 0.15s ease-in;
    transition: all 0.15s ease-in;
  }
  #maeJemison {
    position: relative;
    top: -150px;
    visibility: hidden;
  }
  .speaker-content ul {
    margin: 0 auto;
    width: 90%;
    margin-bottom: 30px;
    text-align: left;
  }
  .speaker-content ul li,
  .speaker-content strong {
    color: #637983!important;
  }
  h3 {
    font-size: 20pt !important;
    line-height: 26pt !important;
  }
  .speaker-quote,
  .speaker-quote img {
    float: left;
    max-width: 100%;
  }
  .speaker-quote h3 {
    margin-top: 30px;
    text-align: left !important;
  }
  .speaker-quote {
    margin-bottom: 20px;
  }
  .quote-quote {
    margin-top: 30px;
  }
  .quote-quote em {
    color: #637983 !important;
  }
  .quote-quote p {
    margin: 0 auto;
    margin-top: 14px;
    margin-bottom: 20px;
    width: 95%;
    float: right;
  }
  .quote-quote h3 {
    font-size: 14pt !important;
    line-height: 20pt !important;
    font-weight: 500;
    margin: 0 auto;
    width: 100%;
    text-align: right !important;
    float: right !important;
  }
  #mae-intro {
    padding: 0;
    width: 100% !important;
  }
  #mae-intro img{ 
    max-width: 260px !important;
    width: 260px !important;
    margin: 0 40px 40px 0;
  }
  #mae-intro h1 {
    margin-bottom: 40px !important;
    font-size: 24pt !important;
    line-height: 30pt !important;
    color: #ffffff !important;
  }
  #mae-intro h3 {
    margin-top: 60px !important;
    font-weight: 700 !important;
  }
  #mae-intro .title-company {
    margin-top: 6px;
    width: 80%;
  }
  #mae-intro .speaker-bio {
    clear: none;
    margin-top: 20px !important;
    font-size: 12pt !important;
    line-height: 16pt !important;
    font-style: italic !important;
  }
  .speaker-quote img {
    max-width: 100%;
    width: 320px;
  }
  .navbar-right {
  display:none;
  }
  .speaker-bio p {
    margin-bottom: 14px;
  }
  .samurai-appear {
    display: none;
  }
  .more-speaker-btn {
    margin: 0 auto;
    text-align: center;
    margin-bottom: 80px;
  }
  .btn-blue {
    will-change: background-color;
    -moz-transition: background-color 0.25s ease-in;
    -webkit-transition: background-color 0.25s ease-in;
    -ms-transition: background-color 0.25s ease-in;
    transition: background-color 0.25s ease-in;
  }
  .interactive-card a {
    color: #2463be !important;
    font-weight: 500;
    will-change: color;
    -moz-transition: color 0.25s ease-in;
    -webkit-transition: color 0.25s ease-in;
    -ms-transition: color 0.25s ease-in;
    transition: color 0.25s ease-in;
  }
  .speaker-info h1 {
    color: #ffffff !important;
    font-weight: 700;
    width: 80%;
  }
  ::-webkit-input-placeholder {
    color: #eeeded;
  }
  ::-moz-placeholder {
    color: #eeeded;
  }
  :-ms-input-placeholder { 
    color: #eeeded;
  }
  :-moz-placeholder { 
    color: #eeeded;
  }
  .mktoForm div, .mktoForm span, .mktoForm label, .mktoForm p {
    margin-left: 0 !important;
  }
  .mktoForm .mktoButtonWrap.mktoSimple .mktoButton {
    margin-top: 2rem !important;
  }
  .speaker-info h3 {
    font-weight: 200;
  }
  #hub18 {
    width: 100% !important;
  }
  .mktoForm .mktoFieldWrap {
    width: 100% !important;
  }
  .mktoForm .mktoFormCol {
    float: left;
    position: relative;
    min-height: 2em;
  }
  .mktoForm, .mktoForm .mktoFormCol {
    margin: 0 auto !important;
    width: 100% !important;
  }
.mktoForm input[type=date], .mktoForm input[type=email], .mktoForm input[type=number], .mktoForm input[type=tel], .mktoForm input[type=text], .mktoForm input[type=url], .mktoForm select.mktoField {
  border-radius: 4px!important;
  background: 0 0!important;
  color: #fff !important;
  width: 100%!important;
}
  select {
    color: #ffffff !important;
  }
  @media screen and (max-width: 992px) {
    #mae-intro .title-company {
      width: 100%;
    }
    .speaker-center {
      float: inherit !important;
    }
    .speaker-quote img {
      float: inherit;
    }
    .ninja-vanish {
      display: none;
    }
    .samurai-appear {
      display: block;
    }
    .speaker-center {
      margin-top: 30px;
    }
    .quote-quote {
      margin-top: 0px;
    }
  }
  @media screen and (max-width: 767px) {
    .speaker-info .title-company {
      margin-top: 6px;
      margin-bottom: 20px;
    }
    .speaker-pic {
      float: none;
    }
    #mae-intro h3 {
      margin-top: 10px !important;
    }
  }
</style>
<div class='site-wrapper'>
 <div class='hero' id='sessions-hero'>
  <div class='container-fluid-bg'>
    <div class='embed-responsive embed-responsive-16by9'>
      <div id='bg-content'>
        <div class='logo-hub17'><img src='https://www.anaplan.com/wp-content/uploads/2016/04/Hub17_Logo_DarkBackground_White-White.png' alt='Hub17' /></div>
        <h1 style='font-size:26px'>Anaplan Conference</h1>
        <p>One platform. One community. Unlimited possibilities.</p>
        <!--<a href='#modal-id-1482188984403' class='colorbox colorbox-full' data-toggle='modal' data-modal-content='https://www.youtube.com/embed/JjdEck3BuBs'><img src='/wp-content/uploads/2016/04/play_button_large_90x90.png' class='play-button'></a>-->
        <p style='margin-top:20px'><a href='https://www.youtube.com/embed/kuzgP-vkdnY' class='colorbox colorbox-full'><img src='/wp-content/uploads/2016/04/play_button_large_90x90.png' class='play-button'/></a></p>
      </div>
      <div id='background-vid'>
        <div id='bg-image'>
          <img src='https://www.anaplan.com/wp-content/uploads/2016/04/Mae_Banner.png' alt='Mae Jemison' />
        </div>
        <!--<div id='true_video_overlay'></div>
        <div id='player'></div>-->
      </div>
    </div>
   </div>
  </div>
  <a id='mae-jemison-trigger' class='animated bounceIn' href='#mae-jemison-intro' data-toggle='modal' data-modal-size='narrow' style='display:none'></a>
  <div class='banner-solid' style='padding: 40px 0;'>
    <div class='container'>
      <div class='row'>
        <div class='col-sm-12 col-md-9 text-center'>
          <a href='/'><span class='logo-anaplan'></span></a>
          [gap height='10']
          <h3>Welcome to a new age of connected planning.</h3>
          [gap height='30']
          <p style='text-align:justify !important;'>Hub17 brought together more than 1,500 members of the global Anaplan community to explore the power of connected planning. With an engaging keynote featuring Dr. Mae Jemison, over 120 breakout sessions for every Anaplanner, a closing-night Street Party, and much more, Hub17 lived up to its motto: One platform. One community. Unlimited possibilities.</p>
          [gap height='30']
          <span style='text-align: center; margin: 0 auto;'>
            <p>Anaplan Hub moves to Las Vegas in 2018! Mark your calendar for our annual conference on March 5–7, and sign up to be notified of early-bird pricing.</p>
            [gap height='10']
            <p style='text-align:center!important;'><a href='#registration-hub18' data-toggle='modal' data-modal-size='narrow' class='btn btn-green btn-register animated bounceIn' style='color: #ffffff !important;'>Save the date</a></p>
          </span>
          </div>
      </div>
    </div>
  </div>
<section id='keynote'>
  <div class='container'>
    <div class='row'>
      <div class='col-sm-12 col-md-9 text-center'>
        <h3>Keynote highlights</h3>
      </div>
    </div>
    [gap height='30']
    <div class='row'>
      <div class='col-sm-12 col-md-6'><a class='speaker keynote animated bounceIn video-playlist' href='#anaplan-vision' data-toggle='modal' data-modal-size='narrow' style='display:none'><span class='playbutton'><img src='https://www.anaplan.com/wp-content/uploads/2016/04/play_button_large_90x90.png' alt='play button' /></span><img src='https://www.anaplan.com/wp-content/uploads/2016/04/Website_thumbnail_61_anaplan-vision_v2.jpg' alt='Anaplan Vision' /></a></div>
      <div class='col-xs-12 col-md-6 keynote-summary'>[gap height='15']<p>Watch the Hub17 keynote highlight video to see inspirational moments from the half-day general session. Anaplan leaders shared the vision of connected planning, introduced the new Anaplan Community, and presented a creative product demonstration, while Anaplan customers and partners discussed how connected planning benefits their businesses and their lives.</p>[gap height='15']</div>
    </div>
    [gap height='10']
    <div class='row'>
      <div class='col-md-4'><a class='speaker keynote animated bounceIn video-playlist' href='#anaplan-customer' data-toggle='modal' data-modal-size='narrow' style='display:none'><span class='playbutton'><img src='https://www.anaplan.com/wp-content/uploads/2016/04/play_button_large_90x90.png' alt='play button' /></span><img src='https://www.anaplan.com/wp-content/uploads/2016/04/Website_thumbnail_62_anaplan-panel.jpg' alt='Anaplan Customer' /></a>
      </div>
      <div class='col-md-4'><a class='speaker keynote animated bounceIn video-playlist' href='#anaplan-community' data-toggle='modal' data-modal-size='narrow' style='display:none'><span class='playbutton'><img src='https://www.anaplan.com/wp-content/uploads/2016/04/play_button_large_90x90.png' alt='play button' /></span><img src='https://www.anaplan.com/hub17/wp-content/uploads/sites/16/2017/04/Website_thumbnail_63_anaplan-community1.jpg' alt='Anaplan Community' /></a>
      </div>
      <div class='col-md-4'><a class='speaker keynote animated bounceIn video-playlist' href='#anaplan-skit' data-toggle='modal' data-modal-size='narrow' style='display:none'><span class='playbutton'><img src='https://www.anaplan.com/wp-content/uploads/2016/04/play_button_large_90x90.png' alt='play button' /></span><img src='https://www.anaplan.com/hub17/wp-content/uploads/sites/16/2017/04/Website_thumbnail_64_anaplan-product-demo.jpg' alt='Anaplan Skit' /></a>
      </div>
    </div>
    <div class='row'>
      <div class='col-sm-12 col-md-12 text-center' style="margin: 40px auto 20px">
        [gap height='10']
        <h3>Breakout sessions</h3>
        [gap height='15']
        <p>A dynamic lineup of 150+ speakers shared stories of how they connected data, people, and plans using the Anaplan platform to spark innovation and deliver business value. Breakout session presenters included Groupon, Intuitive Surgical, Staples, Rockwell Automation, DISH, and dozens more top companies.</p>
        [gap height='10']
      </div>
    </div>
    <div class='row'>
      <div class='col-md-12' id='filter-buttons'>
        <ul>
          <li><a id="all" class='filter-button btn btn-blue btn-outline btn-all' href='#all'>All</a></li>
          <li><a id="sales" class='filter-button btn btn-blue btn-outline' href='#sales'>Sales</a></li>
          <li><a id="finance" class='filter-button btn btn-blue btn-outline' href='#finance'>Finance</a></li>
          <li><a id="supply-chain" class='filter-button btn btn-blue btn-outline' href='#supply-chain'>Supply Chain</a></li>
          <li><a id="hr" class='filter-button btn btn-blue btn-outline' href='#hr'>Workforce</a></li>
          <li><a id="marketing" class='filter-button btn btn-blue btn-outline' href='#marketing'>Marketing</a></li>
          <li><a id="platform" class='filter-button btn btn-blue btn-outline' href='#platform'>Platform</a></li>
        </ul>
      </div>
    </div>
<!-- 
************

RECAP VIDEOS 

************
--> 
${init}
<!--



INSERT VIEW MARKUP HERE




-->
  </div>
</section>
<!--



INSERT MODALS HERE




-->
<div class='modal fade modal-lightbox' id='anaplan-vision' data-youtube='t1NyncdRraU' tabindex='-1' role='dialog' style='display: none; padding-left: 0px;'><div class='modal-dialog modal-full modal-lg' role='document'><div class='modal-content'><div class='modal-body'><div class='embed-responsive embed-responsive-16by9'><iframe src='https://www.youtube.com/embed/t1NyncdRraU?vq=hd1080&amp;modestbranding=1&amp;autoplay=0&amp;iv_load_policy=3&amp;rel=0&amp;showinfo=0&amp;theme=lightwmode=transparent&amp;enablejsapi=1&amp;version=3' webkitallowfullscreen='true' mozallowfullscreen='true' allowfullscreen='true' width='640' height='360' frameborder='0' class='video embed-responsive-item' id='video-anaplan-vision' data-id='anaplan-vision'></iframe></div></div></div></div></div>
<div class='modal fade modal-lightbox' id='anaplan-community' data-youtube='MOYKLIuTQ_w' tabindex='-1' role='dialog' style='display: none; padding-left: 0px;'><div class='modal-dialog modal-full modal-lg' role='document'><div class='modal-content'><div class='modal-body'><div class='embed-responsive embed-responsive-16by9'><iframe src='https://www.youtube.com/embed/MOYKLIuTQ_w?vq=hd1080&amp;modestbranding=1&amp;autoplay=0&amp;iv_load_policy=3&amp;rel=0&amp;showinfo=0&amp;theme=lightwmode=transparent&amp;enablejsapi=1&amp;version=3' webkitallowfullscreen='true' mozallowfullscreen='true' allowfullscreen='true' width='640' height='360' frameborder='0' class='video embed-responsive-item' id='video-anaplan-community' data-id='anaplan-community'></iframe></div></div></div></div></div>
<div class='modal fade modal-lightbox' id='anaplan-customer' data-youtube='F8GbXmmzF4E' tabindex='-1' role='dialog' style='display: none; padding-left: 0px;'><div class='modal-dialog modal-full modal-lg' role='document'><div class='modal-content'><div class='modal-body'><div class='embed-responsive embed-responsive-16by9'><iframe src='https://www.youtube.com/embed/F8GbXmmzF4E?vq=hd1080&amp;modestbranding=1&amp;autoplay=0&amp;iv_load_policy=3&amp;rel=0&amp;showinfo=0&amp;theme=lightwmode=transparent&amp;enablejsapi=1&amp;version=3' webkitallowfullscreen='true' mozallowfullscreen='true' allowfullscreen='true' width='640' height='360' frameborder='0' class='video embed-responsive-item' id='video-anaplan-customer' data-id='anaplan-customer'></iframe></div></div></div></div></div>
<div class='modal fade modal-lightbox' id='anaplan-skit' data-youtube='XmIfr4eD8bc' tabindex='-1' role='dialog' style='display: none; padding-left: 0px;'><div class='modal-dialog modal-full modal-lg' role='document'><div class='modal-content'><div class='modal-body'><div class='embed-responsive embed-responsive-16by9'><iframe src='https://www.youtube.com/embed/XmIfr4eD8bc?vq=hd1080&amp;modestbranding=1&amp;autoplay=0&amp;iv_load_policy=3&amp;rel=0&amp;showinfo=0&amp;theme=lightwmode=transparent&amp;enablejsapi=1&amp;version=3' webkitallowfullscreen='true' mozallowfullscreen='true' allowfullscreen='true' width='640' height='360' frameborder='0' class='video embed-responsive-item' id='video-anaplan-skit' data-id='anaplan-skit'></iframe></div></div></div></div></div>
${modal}
<!--



END INSERT MODALS HERE




-->
    <div class='row'>
      <div class='col-xs-6 col-sm-4 col-md-3 speaker-tile'>
        <a class='speaker animated bounceIn' href='#ray-wang' data-toggle='modal' data-modal-size='narrow' style='display:none'>
          <img src='https://www.anaplan.com/wp-content/uploads/2017/02/RayWang_HUB17_KeyNoteSpeaker_REV2.jpg' alt='' >
          <span class='speaker-name'>Ray Wang</span>
          <span class='speaker-company' style='line-height:16px'>Principal Analyst, Founder, and Chairman</span>
          <img class='speaker-company-logo' src='https://www.anaplan.com/wp-content/uploads/2017/01/hub17-speaker-company-logo-constellation.png' alt=''>
        </a>
      </div>
      <div class='col-xs-6 col-sm-4 col-md-3 speaker-tile'>
        <a class='speaker animated bounceIn' href='#sid-powar' data-toggle='modal' data-modal-size='narrow' style='display:none'>
          <img src='/wp-content/uploads/2017/02/SidPowar_HUB17_KeyNoteSpeaker_REV.jpg' alt='' >
          <span class='speaker-name'>Sid Powar</span>
          <span class='speaker-company' style='line-height:16px'>Senior Manager, Supply Chain Planning & Inventory Control</span>
          <img class='speaker-company-logo' src='/wp-content/uploads/2017/03/dish-logo-grey-150-2.png' alt=''>
        </a>
      </div>
      <div class='col-xs-6 col-sm-4 col-md-3 speaker-tile'>
        <a class='speaker animated bounceIn' href='#andrew-dinwiddie' data-toggle='modal' data-modal-size='narrow' style='display:none'>
          <img src='/wp-content/uploads/2017/02/HUB17_KeyNoteSpeaker_AndrewDinwiddie.jpg' alt='' >
          <span class='speaker-name'>Andrew Dinwiddie</span>
          <span class='speaker-company' style='line-height:16px'>Head of Financial Performance Management</span>
          <img class='speaker-company-logo' src='/wp-content/uploads/2017/02/rsa-logo-220-1.png' alt=''>
        </a>
      </div>
      <div class='col-xs-6 col-sm-4 col-md-3 speaker-tile'>
        <a class='speaker animated bounceIn' href='#jim-miranda' data-toggle='modal' data-modal-size='narrow' style='display:none'>
          <img src='https://www.anaplan.com/wp-content/uploads/2017/02/HUB17_KeyNoteSpeaker_JimMiranda_REV.jpg' alt='' >
          <span class='speaker-name'>Jim Miranda</span>
          <span class='speaker-company' style='line-height:16px'>Global Workforce Planning</span>
          <img class='speaker-company-logo' src='https://www.anaplan.com/wp-content/uploads/2017/02/hub17-speaker-company-logo-kimberly-clark.png' alt=''>
        </a>
      </div>
      <div class='col-xs-6 col-sm-4 col-md-3 speaker-tile'>
        <a class='speaker animated bounceIn' href='#jeff-nelson' data-toggle='modal' data-modal-size='narrow' style='display:none'>
          <img src='/wp-content/uploads/2017/02/hub17-jeff-nelson.jpg' alt='' >
          <span class='speaker-name'>Jeff Nelson</span>
          <span class='speaker-company' style='line-height:16px'>Performance Analytics and Reporting Manager</span>
          <img class='speaker-company-logo' src='https://www.anaplan.com/wp-content/uploads/2017/02/hub17-speaker-company-logo-AFI.png' alt=''>
        </a>
      </div>
      <div class='col-xs-6 col-sm-4 col-md-3 speaker-tile'>
        <a class='speaker animated bounceIn' href='#lalit-singh' data-toggle='modal' data-modal-size='narrow' style='display:none'>
          <img src='/wp-content/uploads/2017/02/LalitSingh_Hub17_speakers-1.jpg' alt='' >
          <span class='speaker-name'>Lalit Singh</span>
          <span class='speaker-company' style='line-height:16px'>Vice President and Software Transformation Program Leader</span>
          <img class='speaker-company-logo' src='https://www.anaplan.com/wp-content/uploads/2017/03/hp-enterprise-logo-150-1.png' alt=''>
        </a>
      </div>
      <div class='col-xs-6 col-sm-4 col-md-3 speaker-tile'>
        <a class='speaker animated bounceIn' href='#stephanie-weiland' data-toggle='modal' data-modal-size='narrow' style='display:none'>
          <img src='/wp-content/uploads/2017/02/StephanieWeiland_HUB17_KeyNoteSpeaker.jpg' alt='' >
          <span class='speaker-name'>Stephanie Weiland</span>
          <span class='speaker-company' style='line-height:16px'>Senior Manager FP&A</span>
          <img class='speaker-company-logo' src='https://www.anaplan.com/wp-content/uploads/2017/02/genoa_logo_150.png' alt=''>
        </a>
      </div>
      <div class='col-xs-6 col-sm-4 col-md-3 speaker-tile'>
        <a class='speaker animated bounceIn' href='#brandon-kulik' data-toggle='modal' data-modal-size='narrow' style='display:none'>
          <img src='/wp-content/uploads/2017/03/HUB17_keynote_BrandonKulik.jpg' alt='' >
          <span class='speaker-name'>Brandon Kulik</span>
          <span class='speaker-company' style='line-height:16px'>Principal: Sales Effectiveness</span>
          <img class='speaker-company-logo' src='https://www.anaplan.com/wp-content/uploads/2017/03/deloitte-logo-grey-150.png' alt=''>
        </a>
      </div>
    </div>
  </div>
</section>
<section id='quotations' class='quotations'>
  <div class='container-fluid'>
    <div class='row'>
      <div class='col-md-12'>
        <div id='quotation-carousel' class='carousel slide' data-ride='carousel'>
          <ol class='carousel-indicators'>
            <li data-target='#quotation-carousel' data-slide-to='0' class='active'></li>
            <li data-target='#quotation-carousel' data-slide-to='1'></li>
            <li data-target='#quotation-carousel' data-slide-to='2'></li>
            <li data-target='#quotation-carousel' data-slide-to='3'></li>
          </ol>
          <div class='carousel-inner' role='listbox'>
            <div class='item active'>
              <div class='quotation'>
                <div class='row'>
                  <div class='col-md-12'>
                    <div class='quote-mark left-quote'></div>
                  </div>
                </div>
                <div class='row'>
                  <div class='col-md-12 quote-content'>
                    <h3>Someone from our team will be back at Hub every year but—to be honest, I hope it’s always me.</h3>
                  </div>
                </div>
                <div class='row'>
                  <div class='col-md-12'>
                    <div class='quote-mark right-quote'></div>
                  </div>
                </div>
              </div>
            </div>
            <div class='item'>
              <div class='quotation'>
                <div class='row'>
                  <div class='col-md-12'>
                    <div class='quote-mark left-quote'></div>
                  </div>
                </div>
                <div class='row'>
                  <div class='col-md-12 quote-content'>
                    <h3>Hub was an incredibly well-organized and informative event. I thoroughly enjoyed it. And it certainly created a buzz around the product and its possibilities.</h3>
                  </div>
                </div>
                <div class='row'>
                  <div class='col-md-12'>
                    <div class='quote-mark right-quote'></div>
                  </div>
                </div>
              </div>
            </div>
            <div class='item'>
              <div class='quotation'>
                <div class='row'>
                  <div class='col-md-12'>
                    <div class='quote-mark left-quote'></div>
                  </div>
                </div>
                <div class='row'>
                  <div class='col-md-12 quote-content'>
                    <h3>As a potential new customer, it was very beneficial. You all put on a really great conference.</h3>
                  </div>
                </div>
                <div class='row'>
                  <div class='col-md-12'>
                    <div class='quote-mark right-quote'></div>
                  </div>
                </div>
              </div>
            </div>
            <div class='item'>
              <div class='quotation'>
                <div class='row'>
                  <div class='col-md-12'>
                    <div class='quote-mark left-quote'></div>
                  </div>
                </div>
                <div class='row'>
                  <div class='col-md-12 quote-content'>
                    <h3>This was my first Hub. It was a great experience! Great content and an awesome opportunity to network with colleagues, customers, and partners.</h3>
                  </div>
                </div>
                <div class='row'>
                  <div class='col-md-12'>
                    <div class='quote-mark right-quote'></div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
<section id='companies'>
  <div class='container'>
    <div class='row'>
      <div class='col-sm-12 col-md-12 text-center speaker-content'>
        [gap height='10']
        <h1>The company we keep</h1>
        [gap height='30']
      </div>
    </div>
    <div class='row'>
      <div class='col-md-12'>
        <ul class='company-logos'>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2016/12/hub17_PG_sm.png' alt='' /></li>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2016/12/hub17_Motorola_sm.png' alt='' /></li>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2016/12/hub17_Del_Monte_sm.png' alt='' /></li>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2016/12/hub17_Verizon_sm.png' alt='' /></li>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2017/03/hub17_bacardi_logo_120.png' alt='' /></li>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2016/12/hub17_Splunk_sm.png' alt='' /></li>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2016/12/hub17_Cisco_sm.png' alt='' /></li>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2016/12/hub17_Docusign_sm.png' alt='' /></li>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2016/12/hub17_HP_sm.png' alt='' /></li>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2016/12/hub17_Kelloggs_sm.png' alt='' /></li>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2016/12/hub17_KC_sm.png' alt='' /></li>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2016/12/hub17_United_sm.png' alt='' /></li>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2016/12/hub17_Intel_sm.png' alt='' /></li>
          <li><img src='https://www.anaplan.com/wp-content/uploads/2016/12/hub17_Groupon_sm.png' alt='' /></li>
        </ul>
      </div>
    </div>
  </div><!-- end Container -->
</section>
<section id='footer-images'>
  <div class='container-fluid'>
    <div class='row'>
      <div class='col-md-6 footer-square width-dir'>
        <img src='https://www.anaplan.com/wp-content/uploads/2016/04/hubwebsite.jpeg' alt='Hub17' />
      </div>
      <div class='col-md-3 footer-square purple-blue'>
        <div class='text-container'>
          <h3>150+</h3>
          <p>expert speakers, including customers, partners, and industry thought leaders</p>
        </div>
      </div>
      <div class='col-md-3 footer-square width-dir-offset'>
        <img src='https://www.anaplan.com/wp-content/uploads/2016/04/20170328-_H7A8756-peter-prato.jpg' alt='Hub17 Snare' />
      </div>
    </div>
    <div class='row'>
      <div class='col-md-3 footer-square green-blue'>
        <div class='text-container'>
          <h3>94%</h3>
          <p>of past attendees would recommend Hub to a colleague</p>
        </div>
      </div>
      <div class='col-md-6 footer-square width-dir width-large'>
        <img src='https://www.anaplan.com/wp-content/uploads/2016/04/20170328-_H7A8792-peter-prato.jpg' alt='Hub17 Audience' />
      </div>
      <div class='col-md-3 footer-square blue-blue'>
        <div class='text-container'>
          <h3>1</h3>
          <p>awesome street party, featuring live music, games, beverage tastings, and more!</p>
        </div>
      </div>
    </div>
</section>
<section id='share' class='share-event'>
  <div class='container' >
    <div class='row'>
      <div class='col-sm-12 col-md-6 sharing-caring right'>
        <h3>Share this event</h3>
      </div>
      <div class='col-sm-12 col-md-6 sharing-caring left'>
        <span class='share-socialmedia'><a class='socialmedia socialmedia-linkedin' href='https://www.linkedin.com/shareArticle?mini=true&amp;url=http%3A//www.anaplan.com/hub17&amp;title=Anaplan%20Hub%202017%20%7C%20San%20Francisco&amp;summary=Come%20learn%20about%20the%20global%20shift%20in%20enterprise%20operations.%20Anaplan%20Hub%20is%20the%20premier%20event%20where%20Anaplanners%20meet,%20learn,%20and%20share.%20Join%20us%20in%20San%20Francisco,%20March%2027-29,%202017.&amp;source='></a> <a class='socialmedia socialmedia-twitter' href='https://twitter.com/home?status=Don't%20miss%20%23AnaplanHub17,%20landing%20in%20San%20Francisco%20on%20March%2027-29.%20Register%20now!%20http%3A//ow.ly/YFYC1%20via%20%40Anaplan'></a> <a class='socialmedia socialmedia-facebook' href='https://www.facebook.com/sharer/sharer.php?u=http%3A//www.anaplan.com/hub17'></a> <a class='socialmedia socialmedia-google' href='https://plus.google.com/share?url=http%3A//www.anaplan.com/hub17'></a></span>
      </div>
    </div>
  </div>
</section>
<div class='modal fade speaker-only' id='registration-hub18' tabindex='-1' role='dialog'>
  <div class='modal-dialog modal-lg speaker-modal-dialog' role='document'>
    <div class='modal-content'>
      <div class='modal-body'>
        <a href='#close' class='speaker-modal-close'></a>
        <div class='speaker-modal-body'>
          <div id='hub18' class='speaker-info'><style>
                .interactive-card a:hover,
                .interactive-card a:active,
                .interactive-card a:focus {
                  color: #000000 !important;
                }
                .mktoForm input[type=text],
                .mktoForm input[type=number],
                .mktoForm input[type=email],
                .mktoForm input[type=url],
                .mktoForm input[type=tel],
                .mktoForm input[type=date],
                .mktoForm select.mktoField {
                  border: 0 !important;
                  border: 1px solid rgba(255,255,255,0.5) !important;
                }
            </style>
            
            <h1 style="font-size: 23pt !important">Anaplan Hub moves to Las Vegas in 2018</h1>
            <h3>Mark your calendar for our annual conference on March 5–7, and sign up to be notified of early-bird pricing.</h3>
            <div class='StandardForm'>
              <script src='//app-abc.marketo.com/js/forms2/js/forms2.min.js'></script>
              <form id='mktoForm_3021'></form>
              <script>MktoForms2.loadForm('//app-abc.marketo.com', '637-RMT-015', 3021);MktoForms2.whenReady(function (form){form.onSuccess(function(values, followUpUrl){form.getFormElem().hide();document.getElementById('confirmform-hub18').style.display = 'block';return false;});});</script>
              <div id='confirmform-hub18' style='display:none;'><p>Thank you. You will receive a confirmation email shortly.</p></div>
            </div> 
            <div style='font-weight: normal; font-size: x-small; padding-top: 20px!important'>
              <a href='https://www.anaplan.com/privacy-policy'>Privacy policy</a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
  </div>
</div>
${scripts}`;
} 

module.exports = sessionsPage;
